awx_collection/plugins/modules/tower_user.py

```diff
@@ -55,6 +55,12 @@
       description:
         - Write-only field used to change the password.
       type: str
+    update_secrets:
+      description:
+        - C(true) will always password if user specifies password and API gives $encrypted$ for password.
```

Comment:
Sentence here is incomplete, looks like it should be "... will always **change** password if user specifies..."